### DISCUSSION TITLE
Grain size gradient

### DISCUSSION THREAD

**Discussion by fzw9381** [neperfepx/neper#456] [Q&A] [answered]
hello everyone!
I want to create a 2D model to carry out my analysis, and a simple example is below. I want to refine the top edge and control the horizontal and vertical size of grains. How can I to do this? Thanks.
![img-n100-gg-id1](https://user-images.githubusercontent.com/56809459/165920695-e3d79330-5991-493c-aa6b-df1177250b54.jpg)

**Comment by rquey** [comment, marked as answer]
See #228.

**Reply by fzw9381** [reply]
Thanks a lot for your reply.
